api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -228,3 +254,54 @@ message Pass {
   // this pass.
   double uplink_center_frequency_hz = 10;
 }
+
+// A plan that specifies antenna configuration for tracking a satellite. A plan
```

Comment:
Is it appropriate to say `antenna configuration` in documentation of an API for satellite operators?

Reply:
Thanks, completely forgot to update this comment.